files/system/qtile/etc/skel/.config/xfce4/helpers.rc

```diff
@@ -0,0 +1 @@
+TerminalEmulator=foot
```

Comment:
This isn't working in qtile for some reason. Thinking maybe we can fix it as a follow up

Reply:
interesting, what's the behavior in thunar?

Reply:
```
Failed to launch preferred application for category "TerminalEmulator".

Could not find fallback TerminalEmulator application.
```

Reply:
In the hyprland image the preferred terminal is kitty, which isn't installed by default in the qtile image. It could also be that some other config somewhere on my machine is incorrect

Reply:
strange. is the skel config actually showing up in ~/.config/xfce4/helpers.rc?

I suspect it might not be because it only has an effect on new users. if you manually copy it in, it will probably fix it

Reply:
Yes, as you say the file isn't overwritten since I already generated it from hyprland. But I already accounted for that. Even if I manually update it to `TerminalEmulator=foot` it still doesn't work -- even with a reboot

Reply:
shame. we could layer xfce terminal as well for qtile as an unbreak?